electrophysiology/models.py

```diff
@@ -70,7 +80,9 @@ class SpikeSorting(BaseExperimentalData):
 
     """
     spike_times = models.ForeignKey(Dataset, blank=True, null=True, related_name="spike_sorting_spike_times",
-                                    help_text="time of each spike relative to experiment start in seconds. TODO: experiment or recording start?")
+                                    help_text="time of each spike relative to experiment start in seconds.")
+									# TODO: experiment or recording start?
+									# Experiment start. Everything should be relative to that.
```

Comment:
The point is that everything within an experiment is on a common timebase. 

Certainly within the .timestamps.npy and spike times files, everything needs to be on the experiment's common timebase : that way, if your recording starts after your eye tracking, the user doesn't need to sync them, it has already been done when the files are created.